Jenkinsfile

```diff
@@ -636,7 +636,70 @@ pipeline{
                   }
                   }
 		}
-            stage('publishAnddhs'){
+        stage('FlexCodeScan'){
+            when {
+                expression{
+                    node('dhmaster'){
+                        props = readProperties file:'data-hub/pipeline.properties';
+                        println(props['ExecutionBranch'])
+                        return (env.BRANCH_NAME==props['ExecutionBranch'])
+                    }
+                }
+            }
+            agent {label 'dhfLinuxAgent'}
+            steps{
+                script {
+
+                    def palamida_url="http://palamida-2.marklogic.com:8888"
+                    def jwt="eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJwYWxhbWlkYSIsInVzZXJJZCI6MywiaWF0IjoxNjA1NTYxMTc5fQ.5NaV_E4FHSe6vJMTX1siZa6f0U-uKm3ssZIfmsPEPzmdkDvtnNxsXRxI1Mj4mHq0i9cBNEHW_ePYKdFh97Sd9Q"
```

Comment:
Add these to Jenkins credentials

Reply:
How to add credentials in Jenkins? I tried but seem like don't have permissions.